src/Symfony/Component/Routing/Tests/Generator/UrlGeneratorTest.php

```diff
@@ -128,6 +128,36 @@ public function testUrlWithNullExtraParameters()
         $this->assertEquals('http://localhost/app.php/testing', $url);
     }
 
+    public function testRelativeUrlWithStringableObjectParameter()
+    {
+        $stringableObject = new StringableObject();
+
+        $routes = $this->getRoutes('test', new Route('/testing/{foo}'));
+        $url = $this->getGenerator($routes)->generate('test', ['foo' => $stringableObject], UrlGeneratorInterface::ABSOLUTE_PATH);
+
+        $this->assertEquals('/app.php/testing/bar', $url);
```

Comment:
assertSame as much as possible

Reply:
done

I keep the others as is, unless we decide where to merge